react-native/index.d.ts

```diff
@@ -5452,7 +5447,7 @@ declare module "react" {
          * This can be used for paginating through children that have lengths smaller than the scroll view.
          * Used in combination with snapToAlignment.
          */
-        snapToInterval?: number
+        snapToInterval?: number[]
```

Comment:
Shouldn't this be a number? An Array here break the build.
https://github.com/facebook/react-native/blob/e974798656eb811036970b72f064dc8a9d10b229/Libraries/Components/ScrollView/ScrollView.js#L309